### DISCUSSION TITLE
Enhancing Anti-Bot Detection Efficacy with SeleniumBase

### DISCUSSION THREAD

**Discussion by benfrankie** [seleniumbase/SeleniumBase#2423] [Q&A] [answered]
Hi Michael, 

I'm currently using SeleniumBase for automating account creation on a particular website. While I've been able to successfully bypass Cloudflare challenges approximately 70% of the time using residential proxies, the accounts just started getting banned rapidly after creation. Manually created accounts using the same proxies remain active, suggesting the platform may have started detecting Selenium usage.

Is SeleniumBase primarily designed for bypassing Cloudflare, or does it also effectively handle other bot detection systems? 
Just wondering if further tweaking SeleniumBase settings would suffice or if I should explore more comprehensive solutions like ZenRows. Please advise. 

Thank you!

**Comment by mdmintz** [comment, marked as answer]
SeleniumBase UC Mode is optimized for any anti-bot detection system that does an actual check for Selenium, such as Cloudflare Turnstile. Other systems such as Google ReCAPTCHA use advanced heuristics for detecting possible automation, which is why they may find false-positives at times. Pretty much all of them track IP addresses and can ban on that. SeleniumBase’s algorithm is currently as advanced as it can be right now (but you may need to adjust timing yourself for optimal behavior). If it’s not optimal enough for your needs, I would suggest trying other solutions.
